clients/pkg/promtail/targets/gcplog/push_target_test.go

```diff
@@ -275,7 +275,15 @@ func TestPushTarget_UseTenantIDHeaderIfPresent(t *testing.T) {
 
 	prometheus.DefaultRegisterer = prometheus.NewRegistry()
 	metrics := gcplog.NewMetrics(prometheus.DefaultRegisterer)
-	pt, err := gcplog.NewGCPLogTarget(metrics, logger, eh, nil, "test_job", config)
+	pt, err := gcplog.NewGCPLogTarget(metrics, logger, eh, []*relabel.Config{
+		{
```

Comment:
nit: could you pull the `[]*relableConfig` into a variable to improve readability of the `NewGCPLogTarget` call?